src/test/scala-2.11/markets/participants/TestLiquidityDemander.scala

```diff
@@ -27,20 +28,21 @@ import scala.concurrent.duration.FiniteDuration
 import scala.concurrent.ExecutionContext.Implicits.global
 
 
-class TestLiquidityDemander(initialDelay: FiniteDuration,
-                            interval: Option[FiniteDuration],
-                            markets: mutable.Map[Tradable, ActorRef],
-                            tickers: mutable.Map[Tradable, Agent[immutable.Seq[Tick]]])
-  extends TestMarketParticipant(markets, tickers)
-  with LiquidityDemander[MarketOrderTradingStrategy] {
+case class TestLiquidityDemander(initialDelay: FiniteDuration,
+                                 interval: Option[FiniteDuration],
+                                 markets: mutable.Map[Tradable, ActorRef],
+                                 tickers: mutable.Map[Tradable, Agent[immutable.Seq[Tick]]])
+  extends LiquidityDemander[MarketOrderTradingStrategy] {
+
+  val outstandingOrders = mutable.Set.empty[Order]
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2091096919/issues/source?bid=2921361&fileBranchId=3140836#l37)